flask/app.py

```diff
@@ -1672,11 +1672,11 @@ def should_ignore_error(self, error):
         """
         return False
 
-    def make_response(self, rv):
+    def make_response(self, retval_from_view_func):
```

Comment:
`view_func_rv`? Not a huge fan of that either but "from" in a variable name usually reeks of too much verbosity